### DISCUSSION TITLE
Is there a sd-webui-controlnet with CPU only support?

### DISCUSSION THREAD

**Discussion by etekinalp** [Mikubill/sd-webui-controlnet#745] [Q&A]
Hello awesome people of the sd-webui-controlnet team,

I'm wondering if there is a way, similar to the sd webui commandline --use-cpu all possiblity, to use only cpu for controlnet as well?

Thank you for any help in advance,

Cheerio
Emre

**Comment by DRFR33MAN24** [comment]
same problem here, does it work with cpu only option or not, I'm trying to running it on vps hosting?

**Comment by newbie-lad** [comment]
Modern “technologies” are not really considerate 😄
Join the world or part ways.

**Comment by lllyasviel** [comment]
CN supports cpu command line

**Reply by etekinalp** [reply]
Many thanks @lllyasviel :D
